template/router/haproxy.cfg.tmpl

```diff
@@ -86,6 +87,7 @@ backend s3-club-7
 
 backend master
         balance leastconn
+        http-response add-header Access-Control-Allow-Origin *
```

Comment:
If the flex frontend uses xhr to talk to flex api this will break things. See the commit message on https://github.com/Financial-Times/s3-club-7/pull/5/commits/f86fe56260e87a56663d355fda8c9d09fa1e7f5e